draft-lmbdhk-quic-multipath.md

```diff
@@ -132,9 +132,11 @@ In addition, we define the following terms:
 The difference between Path Identifier and Packet Number Space Identifier, is that Path Identifier
 is used in multi-path control frames to identify a path, and Packet Number Space Identifier is used
 in 1-RTT packets and ACK_MP frames to distinguish packet number spaces for different paths.
-Both identifiers have the sam value, which is the sequence number of the connection ID, if a
-non-zero connection ID is used. If the connection ID is zero length, the Packet Number Space
-Identifier is 0, while the Path Identifier is selected on path establishment.
+When an endhost uses a non-zero-length Connection ID on a path, the Packet Number Space Identifier
+corresponds to the sequence number of the CID used. In the case one of the hosts uses
+zero-length connection IDs, the packet number space identifier of its path corresponds to the
+sequence number of the Connection ID used by its peer over the same 5-tuple. When both hosts
+uses zero-length connection IDs, the packet number space identifier is 0.
 
```

Comment:
I definitely agree with your first point or to say it more generally, the host opening a new path needs to carry a CID. 

Not entirely sure about the second point: Why do you think it is more complicated to path to a 5-tuple instead of matching the CID? Or do you mean that current offload engines only use the CID and this would simply be a change? Are these kind of offload engines already deployed?

Reply:
The current API only uses the packet itself: look at the packet, find the number context, expand the number to 64 bits, attempt decryption. Using IP addresses makes this a bit more complicated, and introduces new failure modes such as variation of NAT. Using the peer's CID is also error prone. For example, in case of probes, the receiver has no idea of the value of the peer's CID.

See issue #96 for a discussion of options.